docs/sources/flow/get-started/install/puppet.md

```diff
@@ -0,0 +1,105 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/get-started/install/puppet/
```

Comment:
I think we need more aliases than this one. Something like:
```
- /docs/grafana-cloud/agent/flow/get-started/install/puppet/
- /docs/grafana-cloud/monitor-infrastructure/agent/flow/get-started/install/puppet/
- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/get-started/install/puppet/
- /docs/grafana-cloud/send-data/agent/flow/get-started/install/puppet/
```

I'm afraid we've missed it with Chef. Could you add it there too?